src/java/test/org/apache/zookeeper/server/quorum/QuorumPeerMainTest.java

```diff
@@ -413,13 +418,18 @@ public void testBadPeerAddressInQuorum() throws Exception {
         ClientBase.setupTestEnv();
 
         // setup the logger to capture all logs
+        LoggerContext loggerContext =  (LoggerContext) LogManager.getContext(false);
```

Comment:
I am not familiar with Ivy. any idea how to add  maven dependencies with tests qualifier in Ivy?